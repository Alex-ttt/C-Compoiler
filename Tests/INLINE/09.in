inline int fo(){
	return 5;	
}

void main(){
	int a;
	a = fo() + fo();
}