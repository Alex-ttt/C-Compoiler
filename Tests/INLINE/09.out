 Symbols Table:
   fo  int function fo(
  )
    {
      Statements:
      return
        5
--------------------------
    }
   main  void function main(
  )
    {
      Statements:
      {
        Statements:
          inline_temp0
        =
          5
        goto: BLOCK_LABEL2
--------------------------
      BLOCK_LABEL2:
      }
--------------------------
      {
        Statements:
          inline_temp1
        =
          5
        goto: BLOCK_LABEL3
--------------------------
      BLOCK_LABEL3:
      }
--------------------------
        a
      =
          inline_temp0
        +
          inline_temp1
--------------------------
      Symbols Table:
              a int
              inline_temp0 int
              inline_temp1 int
    }
